docs/_includes/confluence.adoc

```diff
@@ -0,0 +1,70 @@
+////
+Header: Convert Confluence XHTML to Asciidoctor
+
+Included in:
+
+- user-manual
+////
+
+You can convert Atlassian Confluence XHTML pages to Asciidoctor using this http://www.groovy-lang.org/download.html[Groovy] script.
+
+The script calls the http://pandoc.org/[Pandoc] tool to convert single or multiple HTML files exported from Confluence to AsciiDoc files (using the Python version of the AsciiDoc toolchain).
```

Comment:
I'll do it tonight. My pro-bono docs tasks are growing at a surprising
cadence.

On Tue, 1 Mar 2016, 16:36 Dan Allen <notifications@github.com> wrote:

> In docs/_includes/confluence.adoc
> <https://github.com/asciidoctor/asciidoctor.org/pull/491#discussion_r54527919>
> :
>
> > @@ -0,0 +1,70 @@
> > +////
> > +Header: Convert Confluence XHTML to Asciidoctor
> > +
> > +Included in:
> > +
> > +- user-manual
> > +////
> > +
> > +You can convert Atlassian Confluence XHTML pages to Asciidoctor using this http://www.groovy-lang.org/download.html[Groovy] script.
> > +
> > +The script calls the http://pandoc.org/[Pandoc] tool to convert single or multiple HTML files exported from Confluence to AsciiDoc files (using the Python version of the AsciiDoc toolchain).
>
> Let me know if you're going to make this update or you need me to do it.
> Don't worry about amending or rebasing at this point. Just push the change
> and I'll make any adjustments necessary to the commits.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/pull/491/files#r54527919>.
>
-- 

Sent from Mobile.